libraries/chain/controller.cpp

```diff
@@ -2340,6 +2340,11 @@ const controller::config& controller::get_config()const
    return my->conf;
 }
 
+const uint32_t controller::get_first_block_num()const
```

Comment:
Why `const uint32_t` ?

Reply:
That is because many functions in controller.cpp has const return value, just keep new function looks similar with other functions.

Reply:
There are many methods on controller that return a `uint32_t`, not a one of them return a `const uint32_t`.

Reply:
Now it is uint32_t,  const is gone. uint32_t don't need const.  using const is to protect a reference return.  really don't need const.